src/deep-di/test/DI.js

```diff
@@ -13,11 +13,15 @@ class FactoryClass {
 }
 
 class ServiceClass {
-  constructor() {
+  constructor(firstDependencyArg, secondDependencyArg, thirdDependencyArg) {
```

Comment:
'thirdDependencyArg' is defined but never used.<br>'secondDependencyArg' is defined but never used.<br>'firstDependencyArg' is defined but never used.